### PR TITLE
Add missing parameters to email_alert class

### PR DESCRIPTION
**Pull request by ajurjevi**

### CODE REVIEW

**Comment by Zenidd** [comment]
Hello @ajurjevi, Thanks for your contribution. 

As soon as our workflow allows us, I'm creating a new branch in our repo with your commits so these get checked against our test. At first glance the variables and template look good.  

Greetings, JP

File: templates/fragments/_email_alert.erb [hunk 1]
```diff
@@ -1,4 +1,27 @@
...
+    <rule_id><%= [@rule_id].flatten.compact.join(',') %></rule_id>
+    <%- end -%>
+    <%- if @do_not_delay -%>
+    <do_not_delay />
+    <%- end -%>
+    <%- if @do_not_group -%>
+    <do_not_group />
+    <%- end -%>
+    <%- if @email_log_source -%>
+    <email_log_source><%= @email_log_source %></email_log_source>
```
Comment (Zenidd) [inline templates/fragments/_email_alert.erb:25]: Hi @ajurjevi,

The [email_log_source](https://documentation.wazuh.com/4.0/user-manual/reference/ossec-conf/global.html#email-log-source) option is a subsetting from [global](https://documentation.wazuh.com/4.0/user-manual/reference/ossec-conf/global.html) section, it shouldn't be part of email_alerts. Would you remove it?

File: templates/fragments/_email_alert.erb [hunk 2]
```diff
@@ -1,4 +1,27 @@
...
-  <email_to><%= @alert_email %></email_to>
-  <% if @alert_group != false -%><group><% @alert_group.each do |gr| -%><%= gr %>,<% end %></group><% end %>
-</email_alerts>
+  <email_alerts>
+    <email_to><%= @alert_email %></email_to>
+    <%- if @level -%>
+    <level><%= @level %></level>
+    <%- end -%>
+    <%- if @alert_group -%>
+    <group><%= [@alert_group].flatten.compact.join(',') %></group>
```
Comment (Zenidd) [inline templates/fragments/_email_alert.erb:7]: Rule groups for [group](https://documentation.wazuh.com/4.0/user-manual/reference/ossec-conf/email_alerts.html#group) options should be separated with a pipe character (“|”).

**Review by Zenidd — CHANGES_REQUESTED** [review]
Hello @ajurjevi, 

First of all, sorry for the late reply. I just reviewed your PR and left some feedback, let me know if I could help with the changes requested. Thank you for your contribution.

_-JP_

File: templates/fragments/_email_alert.erb (hunk 1 above)
Comment (ajurjevi) [reply templates/fragments/_email_alert.erb]: Hi @Zenidd,

please review changes.

Thank you!

File: templates/fragments/_email_alert.erb (hunk 2 above)
Comment (ajurjevi) [reply templates/fragments/_email_alert.erb]: Hi @Zenidd,

please review changes.

Thank you!

**Comment by Zenidd** [comment]
Hi again @ajurjevi,

Everything looks good! I also cloned your branch to check your commits against the `wazuh-puppet` Github actions tests. [Related execution](https://github.com/wazuh/wazuh-puppet/actions/runs/515633684). 

Everything is ok so let's merge it. Thank you for your contribution!


_-JP_

**Review by Zenidd — APPROVED** [review]

**Pull request merged**